pkg/datapath/linux/node_ids.go

```diff
@@ -274,6 +281,16 @@ func (n *linuxNodeHandler) RestoreNodeIDs() {
 	}
 
 	n.registerNodeIDAllocations(nodeIDs)
+	if len(incorrectNodeIDs) > 0 {
+		log.WithFields(logrus.Fields{}).Warnf("Removing %d incorrect node IP to node ID mappings from the BPF map", len(incorrectNodeIDs))
```

Comment:
```suggestion
		log.Warnf("Removing %d incorrect node IP to node ID mappings from the BPF map", len(incorrectNodeIDs))
```
or am I missing something?

Reply:
fixed